ivy/stateful/norms.py

```diff
@@ -151,6 +152,7 @@ def __init__(
         self._bias_init = Zeros()
         self._running_mean_init = Zeros()
         self._running_var_init = Ones()
+        self.training = training
```

Comment:
Actually we wouldn't need to set a `training` variable for `LayerNorm` or `Dropout` explicitly, all this should be handled directly through whether `self.training` was `True` for the parent module (which could of course also be this same module)